pkg/gitserver/gitserver.go

```diff
@@ -50,16 +52,28 @@ ALLOW_LAZY_CREATE
 REQUIRE_GIT_AUTH
   a user/password combination required to access the repo of the form "<user>:<password>"; defaults to none
 REQUIRE_SERVER_AUTH
-	a URL to an OpenShift server for verifying authorization credentials provided by a user. Requires
-	AUTOLINK_NAMESPACE to be set (the namespace that authorization will be checked in). Users must have
-	'get' on 'pods' to pull (be a viewer) and 'create' on 'pods' to push (be an editor)
+  a URL to an OpenShift server for verifying authorization credentials provided by a user. Requires
+  AUTOLINK_NAMESPACE to be set (the namespace that authorization will be checked in). Users must have
+  'get' on 'pods' to pull (be a viewer) and 'create' on 'pods' to push (be an editor)
 GIT_FORCE_CLEAN
   if 'yes', any initial repository directories will be deleted prior to start; defaults to no
   WARNING: this is destructive and you will lose any data you have already pushed
 GIT_INITIAL_CLONE_*=<url>[;<name>]
   each environment variable in this pattern will be cloned when the process starts; failures will be logged
   <name> must be [A-Z0-9_\-\.], the cloned directory name will be lowercased. If the name is invalid the
   process will halt. If the repository already exists on disk, it will be updated from the remote.
+AUTOLINK_KUBECONFIG
+  The location to read auth configuration from for autolinking.
+  If '-', use the service account token to link. The account
```

Comment:
It is a "-"
https://github.com/csrwng/origin/blob/gitserver_bc_controller/pkg/gitserver/autobuild/autobuild.go#L76